samples/snippets/src/main/java/com/example/monitoring/EnableDisableAlertPolicy.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.monitoring;
+
+// [START monitoring_enable_disable_alert_policy]
+import com.google.api.gax.rpc.ApiException;
+import com.google.cloud.monitoring.v3.AlertPolicyServiceClient;
+import com.google.monitoring.v3.Aggregation;
+import com.google.monitoring.v3.AlertPolicy;
+import com.google.monitoring.v3.ComparisonType;
+import com.google.monitoring.v3.UpdateAlertPolicyRequest;
+import com.google.protobuf.BoolValue;
+import com.google.protobuf.Duration;
+import java.io.IOException;
+
+// Sample to enable disable an alert policy
+public class EnableDisableAlertPolicy {
+
+  public static void main(String[] args) throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String displayName = "alert-policy-display-name";
+    String alertPolicyId = "alert-policy-id";
+    // i.e projects/[PROJECT_ID_OR_NUMBER]/alertPolicies/[ALERT_POLICY_ID]
+    enableDisableAlertPolicy(alertPolicyId, false, displayName);
+  }
+
+  public static void enableDisableAlertPolicy(
+      String alertPolicyId, boolean status, String displayName) throws IOException {
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests.
+    try (AlertPolicyServiceClient alertPolicyServiceClient = AlertPolicyServiceClient.create()) {
+      // A Filter that identifies which time series should be compared with the threshold
+      String metricFilter =
+          "metric.type=\"compute.googleapis.com/instance/"
+              + "cpu/utilization\" AND resource.type=\"gce_instance\"";
```

Comment:
All this quoting is difficult to follow, perhaps there is a better way to express this?  String.Format or Stirng.Join combined w/ String.Format ??

Reply:
Done